velox/exec/tests/TableScanTest.cpp

```diff
@@ -1388,12 +1388,26 @@ TEST_F(TableScanTest, splitOffsetAndLength) {
 }
 
 TEST_F(TableScanTest, fileNotFound) {
-  CursorParameters params;
-  params.planNode = tableScanNode();
-
-  auto cursor = TaskCursor::create(params);
-  cursor->task()->addSplit("0", makeHiveSplit("/path/to/nowhere.orc"));
-  EXPECT_THROW(cursor->moveNext(), VeloxRuntimeError);
+  auto split = HiveConnectorSplitBuilder("/path/to/nowhere.orc").build();
+  for (bool ignoreMissingFiles : {true, false}) {
```

Comment:
nit: perhaps, it would be more readable as 

```
assertMissingFile(true);

VELOX_ASSERT_RUNTIME_CODE_THROW(
          assertMissingFile(false),
          error_code::kFileNotFound);
```

without the loop

Reply:
Updated, thanks.